site/src/docs/components/header/code.mdx

```diff
@@ -304,7 +304,95 @@ import { Header, IconSearch } from 'hds-react';
+
+```jsx
+import { Header, IconSearch } from 'hds-react';
+
+() => (
+  <Header theme={{
+    '--color-focus-outline': 'var(--color-metro-dark)',
+    '--header-color': 'var(--color-black)',
+    '--actionbar-background-color': 'var(--color-engel)',
+    '--nav-background-color': 'var(--color-engel-light)',
+    '--nav-border-color': 'var(--color-black)',
+    '--nav-link-hover-color': 'var(--color-black)',
+    '--universal-bar-background-color': 'var(--color-black-20)',
+    '--nav-link-dropdown-background-color': 'var(--color-white)',
+    '--nav-button-background-color': 'var(--color-black-20)',
+    '--nav-button-hover-background-color': 'var(--color-black-40)',
+    '--nav-drop-down-icon-color': 'var(--color-black)',
+    '--header-background-color': 'transparent',
+  }}>
+    <Header.SkipLink skipTo="#content" label="Skip To Content" />
+    <Header.UniversalBar primaryLinkHref="#" primaryLinkText="Helsingin kaupunki">
+      <Header.Link href="#" label="Uutiset" />
+      <Header.Link href="#" label="Asioi verkossa" />
+      <Header.Link href="#" label="Anna palautetta" />
+    </Header.UniversalBar>
+    <Header.ActionBar
+      title="Helsingin kaupunki"
+      titleAriaLabel="Helsingin kaupunki"
+      titleHref="https://hel.fi"
+      logoAriaLabel="Service logo"
+      logoHref="https://hel.fi"
+      menuButtonAriaLabel="Menu"
+    
+    >
+      <Header.LanguageSelector
+        languages={[
+          { label: 'Suomi', value: 'fi' },
+          { label: 'Svenska', value: 'sv' },
+          { label: 'English', value: 'en' },
+        ]}
+      />
+      <Header.ActionBarItem label="Haku" icon={<IconSearch />} id="action-bar-search">
+        <Header.Search label="Hae palvelusta" onChange={() => {}} onSubmit={() => {}} />
+      </Header.ActionBarItem>
+    </Header.ActionBar>
+    <Header.NavigationMenu>
+      <Header.Link
+        as={Link}
+        href="#"
+        label="Sosiaali- ja terveyspalvelut"
+        onClick={(event) => event.preventDefault()}
+        active
+        dropdownLinks={[
+          <Header.Link
+            as={Link}
+            href="#"
+            label="Terveydenhoito"
+            active
+            dropdownLinks={[
+              <Header.Link as={Link} href="#" label="Hammashoito" />,
+              <Header.Link as={Link} href="#" label="Julkinen terveydenhoito" />,
+            ]}
+          />,
+          <Header.Link
+            as={Link}
+            href="#"
+            label="Senioripalvelut"
+            dropdownLinks={[
+              <Header.Link as={Link} href="#" label="Viriketoiminta" />,
+              <Header.Link as={Link} href="#" label="Kuntouttavat palvelut" />,
+            ]}
+          />,
+        ]}
+      />
+      <Header.Link
+        as={Link}
+        href="#"
+        label="Kasvatus ja koulutus"
+      />
+      <Header.Link as={Link} href="#" label="Asuminen" />
+    </Header.NavigationMenu>
+  </Header>
+)
+```
+
+</Playground>
```

Comment:
This piece should also have the Mikko Jämiä Special comment below it.

Reply:
fixed

Reply:
The link points to dark theme, not custom theme.

Reply:
fixed :D